doc/explanation/storage.md

```diff
@@ -122,7 +122,7 @@ Storage volumes can be of the following types:
 : LXD automatically creates one of these storage volumes when you launch an instance.
   It is used as the root disk for the instance, and it is destroyed when the instance is deleted.
 
-  This storage volume is created in the storage pool that is specified in the profile used when launching the instance (or the default profile, if no profile is specified).
+  This storage volume is created in the storage pool that is specified when launching the instance (or in the default profile, if no pool or profile is specified).
```

Comment:
```suggestion
  This storage volume is created in the storage pool that is specified when launching the instance. This pool can be specified explicitly or by specifying a profile. If no pool or profile is specified, LXD uses the storage pool of the default profile.
```

Reply:
I find this suggested version to be easier to understand because it explains that the storage pool can be specified explicitly or via the profile. The current version only implies (in "if no pool or profile is specified") that a profile can be used to specify the pool.

Without this edit, it's also difficult to parse what clause the parenthesized content modifies--that is, what does the part following "**or**" replace? I know after reading it a couple times that it's meant to replace "when launching the instance". However, at first glance, it's natural to parse it as "This storage volume is created in the default profile, if no pool or profile is specified" due to the symmetry of "**in the storage pool**" and "**in the default profile**", and that doesn't make sense.

Reply:
I totally agree RE clarity. Having looked at this again today I'm not sure that this is the right spot in the docs to nail down the semantics of which pool is selected for a root disk device at all; the [section above](https://documentation.ubuntu.com/lxd/en/latest/explanation/storage/#default-storage-pool) is probably more appropriate.

I'm wondering if the relationship between the root disk device and the `container`/`virtual-machine` volume type is a bit of a docs gap; figuring out the relationship between those two took me some time. Probably out of scope for this PR.